Source/Bricks/Collection/CollectionBrick.swift

```diff
@@ -207,8 +207,9 @@ extension CollectionBrickCell: BrickLayoutDelegate {
             return
         }
 
-        sizeChangedHandler?(self)
-        brickCollectionView.layoutSubviews()
+        self.resizeDelegate?.performResize(cell: self, completion: { [weak self] (completed: Bool) in
```

Comment:
I actually like to always see the parameter so I know what it is, even if it's not used. Otherwise I have to look elsewhere if I even want to use it, slightly quicker IMO, but either way.